docs/testplan/PFC_Snappi_Additional_Testcases.md

```diff
@@ -0,0 +1,65 @@
+|           06 | CONGESTION (MACSEC)      | **CONGESTION DETECTION WITH PAUSE FRAMES OVER MACSEC ENABLED LINK:** <br>Purpose of the test is to ensure that congestion is detected on DUT egress with MACSEC enabled links on ingress and egress and mix of lossless and lossy traffic. No drops for lossless priority traffic.                                        | - Single 100Gbps Ingress and Egress - PFC over MACSEC. <br> - Repeat with 400Gbps Ingress and Egress <br> - 1024B packet-size                                                                                                          | - 90% - Lossless Priority 3 and 4 per ingress LC <br>- 10% - Lossy Background Flow [Priority 0, 1, and 2] <br>- Disable PFCWD before the test <br>- Send PAUSE frames to egress interface of DUT.                                                                      | - Single LC - Multiple ASIC <br>- Single LC - Single ASIC <br>- Multiple LC      | 1. There should be NO packet drops on the DUT (IXIA Tx packets == IXIA Rx packets)<br>2. There should be PFCs on IXIA and DUT - Tx and Rx port. <br>3. No Lossless Priority traffic on IXIA Rx Interface. <br> 4. Only 10% lossy priority traffic allowed from the DUT. <br>5. No loss for Lossless traffic.                                                                                                                                                                                   |
+|           07 | CONGESTION (MACSEC)      | **CONGESTION OVER MISMATCHED INGRESS AND EGRESS MACSEC ENABLED LINK:**<br>Test case to check if the DUT detects congestion due to mismatched MACSEC enabled link speeds and sends PFCs to rate-limit the lossless priority traffic.                                                                                       | - Single 400Gbps Ingress and single 100Gbps egress <br> - Repeat with IPv6 <br>- 1024B packet-size.                                                                                                                                    | - 90% - ingress line rate Priority 3 and 4 <br>- 10% of ingress line-rate -- Background Flow[Priority 0, 1, and 2]                                                                                                                                                     | - Single LC - Multiple ASIC <br>- Multiple LC                                    | 1. There should be NO packet drops on the DUT for both Lossless and lossy priority traffic.<br>2. DUT should send PFCs to the traffic generator to rate-limit the lossless priority traffic.<br>3. 100Gbps Egress port should receive Lossless Priority 3 and 4 traffic.                                                                                                                                                                                                                       |

```

Comment:
MACsec with PFC is not yet supported in SW/HW on Ixia. Looking for the feasibility of implementing this feature.